app/services/service_toggles.py

```diff
@@ -46,8 +48,18 @@ def get_business_details(self, ru):
     def get_user_details(self, uuid):
         return self._service.get_user_details(uuid)
 
-"""party is the interface that code should use mocktoggle for the party service """
-party = Party(False)
 
+class Case(ServiceMockToggle):
+    """A case service acts as an interface to mocked or real Case Services via its ServiceMockToggle base"""
+
+    def __init__(self, use_mock=False):
+        super().__init__(use_mock, CaseService, CaseServiceMock, 'Case')
 
+    def store_case_event(self, case_id, user_uuid):
+        return self._service.store_case_event(case_id, user_uuid)
+
+"""party is the interface that code should use mocktoggle for the party service """
+party = Party(False)
 
+"""case_service is how code should interact with the case service"""
+case_service = Case(False)
```

Comment:
I am open on this one , and can see both sides of the argument . Right now I dont think an environment variable is the correct way to go as we will never have a production use case where we need to use the mock . Its only of use for developer testing and running unit tests . So an environment variable/config is a risk for little benifit.